miso-web/src/main/resources/external/miso.properties

```diff
@@ -39,6 +39,10 @@ miso.alerting.enabled:true
 # toggle mirroring of all alerts to system alerts (user id 0)
 miso.alerting.saveSystemAlerts:true
 
+## configs for caching
+# precache objects at MISO startup. This will greatly increase startup time, but will improve MISO performance for users at the outset
+miso.db.caching.precache.enabled:false
```

Comment:
This won't do much since Hibernate takes care of the caching, but that means it can be deleted later :)